crates/fluvio/src/producer/partition_producer.rs

```diff
@@ -210,8 +210,11 @@ impl PartitionProducer {
         }
 
         topic_request.partitions.push(partition_request);
-        request.acks = 1;
-        request.timeout_ms = 1500;
+        request.acks = match self.config.isolation {
```

Comment:
Maybe we should change acks to Rust enum instead of doing conversion here.

Reply:
It will break Kafka wire protocol. Do we need to follow it? If no, then enum looks better here, I agree.

Reply:
We don't have to follow Kafka wire protocol.  You can still keep protocol compatible but use Rust Enum by just conforming to the same encoding and decoding behavior.  The only downside is bumping up the minor version number but I think it's worthwhile.  We should migrate legacy C or Kafka-style API to Rust native

Reply:
Done